ee/be/modules/600-flant-integration/images/flant-pricing/hooks/node_group_resources_metrics.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash -e
+
+# Copyright 2024 Flant JSC
+# Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+
+for f in $(find /frameworks/shell/ -type f -iname "*.sh"); do
+  source $f
+done
+
+function __config__() {
+  cat << EOF
+    configVersion: v1
+    kubernetes:
+    - name: ngs
+      group: ngs
+      keepFullObjectsInMemory: false
+      queue: /node_group_resources_metrics
+      apiVersion: deckhouse.io/v1
+      kind: NodeGroup
+      jqFilter: |
+        {
+          "name": .metadata.name,
+          "labels": .spec.labels,
+          "taints": .spec.taints
```

Comment:
I believe that `.spec.taints` and `.spec.labels` is a mistake. It should be at lease `.spec.nodeTemplate`.

Reply:
Yes, this is a mistake.